src/util/diproche/predicateList.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
I think this json would become unnecessary. That it needs to have the keys listed twice makes is very unconvient to edit and add to as well.